host/testdata/matching_simulation_fluctuating_adaptive.yaml

```diff
@@ -0,0 +1,48 @@
+enablearchival: false
+clusterno: 1
+messagingclientconfig:
+  usemock: true
+historyconfig:
+  numhistoryshards: 4
+  numhistoryhosts: 1
+matchingconfig:
+  nummatchinghosts: 4
+  simulationconfig:
+    tasklistwritepartitions: 0 # this doesn't matter. adaptive scaler will start from 1
+    tasklistreadpartitions: 0 # this doesn't matter. adaptive scaler will start from 1
+    forwardermaxoutstandingpolls: 1
+    forwardermaxoutstandingtasks: 1
+    forwardermaxratepersecond: 10
+    forwardermaxchildrenpernode: 20
+    localpollwaittime: 10ms
+    localtaskwaittime: 10ms
+    tasks:
+      - numtaskgenerators: 3
```

Comment:
Nit: We'd talked a bit about changing the `numtaskgenerators` field since it's somewhat error prone. Can 3 task generators actually dispatch 250 tasks/sec? If it can I think we should remove the field entirely and just use a constant value of like 10, since that would always be more than enough.

Reply:
I don't see a need to expose this as it doesn't add value to simulation scenarios. It's just another thing that can be bottleneck in a confusing way. I will leave it for now but let's do a cleanup of this field and use a fixed value as discussed.